airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -242,24 +242,34 @@ def post_dag_run(dag_id, session):
     except ValidationError as err:
         raise BadRequest(detail=str(err))
 
+    try:
+        schedule_date = post_body.pop("schedule_date")
+    except KeyError:
+        schedule_date = post_body.pop("execution_date")
+
```

Comment:
```suggestion
   schedule_date = post_body.get("schedule_date", post_body["execution_date"])
```
This way we preserve the content of post_body. Or what do you think?

Reply:
`DagRun` does not accept `schedule_date` as a keyword argument (yet), so either we need to modify `post_body` or we can’t use `DagRun(..., **post_body)`.